gammapy/background/tests/test_energy_offset_array.py

```diff
@@ -1,21 +1,51 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
 from __future__ import absolute_import, division, print_function, unicode_literals
+from numpy.testing import assert_equal
 import numpy as np
-from ..energy_offset_array import EnergyOffsetArray
 from ...data import DataStore
 from ...datasets import gammapy_extra
 from ...utils.testing import requires_dependency, requires_data
 from ...utils.energy import EnergyBounds
+from ..energy_offset_array import EnergyOffsetArray
+
+
+def make_test_array():
+    ebounds = EnergyBounds.equal_log_spacing(0.1, 100, 20, 'TeV')
+    offset = np.linspace(0, 2.5, 10)
+    array = EnergyOffsetArray(ebounds, offset)
+    # put some dummy example data
+    # TODO: this could be made optional or split out into a
+    # separate utility function later on when lookup / interpolation methods are added to the class.
+    array.data[5, 3] = 42
```

Comment:
One of the methods that should be added to this class is `.evaluate(energy, offset)` that gives the interpolated array value at a given energy and offset.
A test for this could use nearest-neighbor interpolation and check that the value 42 is obtained.

Although ... thinking about this now .... there are better ways to create a test case for this.
So if you like, you can remove that line and comment for now.

OK?